packages/address/src/address.ts

```diff
@@ -30,17 +30,24 @@ import {
  */
 export default class Address extends AbstractAddress {
   // #region address-2
+  /**
+   * @deprecated
+   * Type `Bech32Address` is now deprecated, as is this property. Use `B256` addresses instead. ([help](https://docs.fuel.network/docs/specs/abi/argument-encoding/#b256))
+   */
   readonly bech32Address: Bech32Address;
   // #endregion address-2
 
   /**
    * @param address - A Bech32 address
```

Comment:
```suggestion
   * @param address - A B256 address
```

Reply:
This constructor now accepts both types, so I've updated the language as such 0568c2955